portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -23,6 +23,7 @@ definition {
 		Pause(locator1 = "2000");
 
 		Click.javaScriptClick(
+			key_fieldLabel = "${fieldLabel}",
```

Comment:
@rijema I noticed there are other places using this macro, like `FormsUploadField#AllowsFiveUploadsWhenRepeatableIsEnabled`, `FormsColorField#CanBeSetToRepeatable`, `FormsFieldsGroup#CanBeRepeatable`, and a few others. 

Why didn't you update these tests?

Reply:
Hi @carolmariaabb , I noticed these tests before, but my unfortunately I assumed that I only had to change the tests that were failing. I just update this pr with the other changes.